server/services/translation_service.py

```diff
@@ -0,0 +1,58 @@
+import re
+import glob2
+
+
+class TranslationService:
+
+    @staticmethod
+    def refresh_translatables():
+        """
+        Read through frontend files and extract translatable strings
+        """
+
+        # Get translatable strings
+        strings = TranslationService._get_translatables()
+
+        # Replace english language file, which is the source for translation
+        output = open('client/locale/en.json', 'w')
+        output.write('{\n')
+        counter = len(strings)
+        for string in sorted(strings):
+            if counter > 1:
+                output.write('\t"%s": "%s",\n' % (string, string))
+                counter = counter - 1
+            else:
+                output.write('\t"%s": "%s"\n' % (string, string))
+        output.write('}\n')
+        output.close()
+
+    @staticmethod
+    def _get_translatables():
+
+        # Source files containing translation strings
+        files = glob2.glob('client/app/**')
+        files.append('client/index.html')
+
+        # Regular expression to detect translation strings
+        translate_string = re.compile('\{\{\s+[\'\"]([^\|]*)[\'\"]\s+\|\s+translate\s+\}\}')
```

Comment:
My main suggestion would be to change `+`s to `*`s here as there seem to be a few translatable strings without the prettified spacing. I guess ideally we would be consistent across all the strings about the spacing, so I can also see the case to not include that--we'll just need to fix some of those inconsistencies (e.g. [[1](https://github.com/hotosm/tasking-manager/blob/14c4d5885ad912c688391e15ede24cd7e6f31602/client/app/about/faq.html#L19)][[2](https://github.com/hotosm/tasking-manager/blob/659ee5b9960ced26b04042c88d393ca5a5c3da17/client/app/about/about.html#L15)] and about 40 others).

Reply:
Thanks, @ethan-nelson! Adjusted as you suggested.